src/models/UniqueNamedContainer.java

```diff
@@ -107,20 +100,20 @@ private Optional<T> findFirstWithUniqueName(String uniqueName) {
      * @param originalName The original name to look for.
      * @return An optional item which is empty if an item with the original name could not be found.
      */
-    public Optional<T> findFirstWithOriginalName(String originalName) {
+    public Optional<T> findAnyWithOriginalName(String originalName) {
         return items.stream().filter(item -> Objects.equals(item.getOriginalName(), originalName)).findFirst();
     }
 
     /**
-     * @return The internal list representation of this container.
+     * @return the internal list representation of this container.
      */
```

Comment:
Also add a normal description, i.e., a documentation comment should not just consist of `@` fields.